.github/workflows/gh-pages.yml

```diff
@@ -8,6 +8,7 @@ on:
         description: Which branch to deploy to peggyjs.org
         required: true
         options:
+          - main
```

Comment:
The `gh-pages` workflow might be deleted after we fully switch over.  I want to leave it in place for the short term so that I can easily re-publish just the docs if something goes wrong.

The input is provided when you run the gh-pages workflow manually.  The Actions section of the GitHub repository provides a mechanism for running the action and for gathering the input from the person requesting the action be run.